custom_components/tuya_local/devices/sefaul_q9_dehumidifier.yaml

```diff
@@ -0,0 +1,68 @@
+name: Dehumidifier
+products:
+  - id: e8c9gukirorvyhk8
+    manufacturer: Sefaul
+    model: Q9 Dehumidifier
+
+entities:
+  - entity: light
+    class: light
+    name: Light
+    dps:
+      - id: 11
+        name: switch
+        type: boolean
+  - entity: fan
+    class: fan
+    name: Purifier
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 4
+        type: string
+        name: preset_mode

```

Comment:
If you use `translation_key: fan_with_presets`, then change these low and high to all lower case the values will get translated.